server/lib/base_konnector.js

```diff
@@ -32,6 +34,7 @@ module.exports = {
       slug: slug,
       description: `konnector description ${slug}`,
       logger: logger,
+      docTypeVersion: docTypeVersion,
```

Comment:
It's to help developers who will re-use these data to understand what field they could expect on the data they get from the database of the Cozy. 
The main idea is: the app which create one document write there its name and version, so a future developer want to really know each details about the format, they can go on github and directly check the code which emit it.